element_deeplabcut/model.py

```diff
@@ -3,6 +3,7 @@
 MIT License Copyright (c) 2022 Mackenzie Mathis
 DataJoint Schema for DeepLabCut 2.x, Supports 2D and 3D DLC via triangulation.
 """
+from logging import root

```

Comment:
What uses this import? I suspect VS' put that here.

Reply:
I've seen that recently - vscode will just assume a package for something. How do I prevent this?

Reply:
No idea. It happened to me as well, multiple times....